config/taxonomy/example/module/capability.yaml

```diff
@@ -0,0 +1,34 @@
+definitions:
```

Comment:
Is that really just an example?

Reply:
@eletonia suggest we move it to the base taxonomy. I expect that these fields are required.
I am not sure about the values of capabilities, maybe @froesef or @simanadler can comment if they can change or needs to be fixed.

Reply:
@roee88 @ronenkat Ok I deleted the `capability.yaml` example and added the `Capability` definition to `base.yaml`